src/layouts/base-new/index.tsx

```diff
@@ -40,7 +44,12 @@ const BaseNewLayout: React.FC<BaseNewLayoutProps> = ({
           )}
         </div>
       </CoreDevDotLayout>
-      <ConsentManager />
+      <ConsentManager
+        onAcceptAll={() => {
+          handleOptInAnalytics(query['optInFrom'] as OptInPlatformOption)
+          console.log('accepted!!')
```

Comment:
TODO - it doesn't seem like this function is firing as expected. I need to debug within the consent manager 🤔

Reply:
I got curious and checked this out myself, it looks like you have to pass the props through the object that's passed to `createConsentManager`! I tried it out and that worked for me. Although, I was expecting your approach here to work as well, and it feels weird that it doesn't work.

Reply:
Oh interesting! Okay I'll try that. Ty 😄

Reply:
Sure thing! 😊